components/eamxx/scripts/machines_specs.py

```diff
@@ -57,6 +57,10 @@
                 ["mpicxx","mpifort","mpicc"],
                 "bsub -I -q batch -W 0:30 -P cli115 -nnodes 1",
                 "/gpfs/alpine/cli115/proj-shared/scream/master-baselines"),
+    "pm-cpu" : (["module purge", "module use --append /global/cfs/cdirs/e3sm/eamxx-ml/install/install/modulefiles", "module load PrgEnv-gnu gcc-native/12.3 cray-libsci craype cray-mpich cray-hdf5-parallel cray-netcdf-hdf5parallel cray-parallel-netcdf cmake evp-patch cray-python/3.9.13.1 screamML-venv/0.0.1", "export LD_LIBRARY_PATH=/opt/cray/pe/python/3.9.13.1/lib:/global/common/software/nersc/pm-2022q4/sw/evp-patch/lib64:/opt/cray/libfabric/1.15.2.0/lib64:/usr/lib64/gcc/x86_64-suse-linux/12"],
```

Comment:
I believe there's a bug in Perlmutter's `LD_LIBRARY_PATH`, or perhaps I'm loading an extra module that I shouldn't be. When using `gcc-native/12.3`, it should look under `/usr/lib64/gcc/x86_64-suse-linux/12` but `/opt/cray/pe/gcc-libs` keeps showing up in `LD_LIBRARY_PATH` despite module purge first. This is the error:
```
/opt/cray/pe/gcc-libs/libstdc++.so.6: version `GLIBCXX_3.4.30' not found
/opt/cray/pe/gcc-libs/libgcc_s.so.1: version `GCC_12.0.0' not found
```

As a workaround, I'm manually overwriting `LD_LIBRARY_PATH` to remove `/opt/cray/pe/gcc-libs`. Either we fix this by finding out which module causes this path to be added and removing that, or we should open a ticket to ask support if this is an unexpected behavior.